miso-web/src/main/webapp/scripts/list_runaliquot.js

```diff
@@ -66,19 +108,37 @@ ListTarget.runaliquot = {
     }, "aliquotAlias", 0, true), {
       sTitle: "QC Status",
       mData: "qcStatusId",
-      mRender: ListUtils.render.textFromId(Constants.runLibraryQcStatuses, 'description', 'Pending')
+      mRender: function(data, type, full) {
+        if (data === undefined || data === null) {
+          return 'Pending';
+        }
+        var status = Utils.array.findUniqueOrThrow(Utils.array.idPredicate(data), Constants.runLibraryQcStatuses);
+        if (type !== 'display') {
+          return status.description;
+        }
+        return '<div class="tooltip"><span>' + status.description + '</span>'
+            + '<span class=tooltiptext>Set by ' + full.qcUserName + ', ' + full.qcDate
+            + '</span></div>';
+      }
     }, {
       sTitle: "QC Note",
       mData: 'qcNote',
       sDefaultContent: ''
     }, {
-      sTitle: 'QC User',
-      mData: 'qcUserName',
-      sDefaultContent: 'n/a'
-    }, {
-      sTitle: 'QC Date',
-      mData: 'qcDate',
-      sDefaultContent: 'n/a'
+      sTitle: 'Data Review',
+      mData: 'dataReview',
+      mRender: function(data, type, full) {
+        if (data === undefined || data === null) {
+          return 'Pending';
+        }
+        var status = data ? 'Pass' : 'Fail';
+        if (type !== 'display') {
+          return status;
+        }
+        return '<div class="tooltip"><span>' + status + '</span>'
+            + '<span class=tooltiptext>Set by ' + full.dataReviewer + ', ' + full.dataReviewDate
```

Comment:
Inconsistent quotation on class attribute of div and span?

Reply:
Fixed